MMM-TeslaFi.js

```diff
@@ -281,5 +307,25 @@ Module.register('MMM-TeslaFi', {
 		} else {
 			return this.numberFormat(valueMiles) + " miles";
 		}
+	},
+
+	// Converts given speed (assumes miles input) to configured output with approprate units appened
+	convertSpeed: function(valueMiles) {
+		if(this.config.unitDistance==="km") {
+			return this.numberFormat(valueMiles * 1.60934) + " Km/h"
```

Comment:
Could you change this to the correct SI unit of "km/h" rather than "Km/h"?

Reply:
done, good catch